include/rosdiscover-clang/RawStatement.h

```diff
@@ -20,6 +25,99 @@ class RawStatement {
   virtual RawStatementKind getKind() = 0;
 };
 
+class RawCompound: public RawStatement {
```

Comment:
nit pick: missing space between classname and colon

Reply:
fixed